tests/test_packages/test_skills_integration/test_coin_price.py

```diff
@@ -89,7 +89,7 @@ def test_coin_price(self):
         self.run_install()
 
         process = self.run_agent()
-        is_running = self.is_running(process)
+        is_running = self.is_running(process, timeout=30)
```

Comment:
why so long?

Reply:
This was a check for what seems to be a flaky test, but it looks fine now so I reverted back to default timeout.